.github/workflows/ci.yml

```diff
@@ -218,12 +218,11 @@ jobs:
 
       - uses: actions/checkout@v4
 
-      # NOTE(aznashwan): starting with Golang 1.21, the windows-2019 GitHub runner's
-      # builtin MinGW version leads to DLL loading errors during runtime.
-      - name: Upgrade MinGW on Windows 2019
+      # needs to be a separate step since terminal reload is required to bring in new env variables and PATH
+      - name: Upgrade Chocolaty on Windows 2019
         if: matrix.os == 'windows-2019'
+        shell: powershell
         run: |
```

Comment:
just a question: this is an empty step (which maybe makes sense with the comment about "terminal reload") but the name of the step is still "Upgrade Chocolaty on Windows 2019"; that may be confusing to future contributors trying to figure out what this does--if this is needed for a shell reload, can we change the step name to something like that so someone doesn't remove it in the future thinking it's a mistake.

Reply:
Yes, this is a noop. Got rid of it.

Reply:
Great! thank you